nagiosplugin/state.py

```diff
@@ -2,8 +2,7 @@
 """Classes  to represent check outcomes.
 
 This module defines :class:`ServiceState` which is the abstract base
-class for check outcomes. class for check outcomes. class for check
-outcomes. The four states defined by the :term:`Nagios plugin API` are
+class for check outcomes. The four states defined by the :term:`Nagios plugin API` are
```

Comment:
The line wrap here violates PEP8.  Could you run this through Flake8 or just re-wrap the paragraph to 78 characters?

Reply:
The file `HACKING.txt` describes how to set up the test environment so that `tox` can run thee tests for you before you even stage for commit.